spec/menu_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'menu'
+require 'goods'
+
+describe Menu do
+  let(:food1) { double :food, to_s: "Item #1: Nachos - £6 (Starter)", class: Food, id: 1 }
+  let(:food2) { double :food, to_s: "Item #2: Pizza - £6 (Main)", class: Food, id: 2 }
+  let(:food3) { double :food, to_s: "Item #3: Chips - £3 (Side)", class: Food, id: 3 }
+  let(:food4) { double :food, to_s: "Item #4: Cake - £3 (Dessert)", class: Food, id: 4 }
+  let(:drink1) { double :drink, to_s: "Item #5: Beer - £4 (Drink)", class: Drink, id: 5 }
+  let(:sample_menu) { Menu.new([food1, food2, food3, food4, drink1]) }
+
+  it "can be initialised with an array of items" do
+    Menu.new([food1, food2])
+  end
+
+  describe "#display_all" do
+    it "calls #to_s on every item in @items" do
+      expect(food1).to receive(:to_s)
+      expect(food2).to receive(:to_s)
+      expect(food3).to receive(:to_s)
+      expect(food4).to receive(:to_s)
+      expect(drink1).to receive(:to_s)
+      expect { sample_menu.display_all }.to output.to_stdout
```

Comment:
What if it prints "hello world" to std_out - would the test pass? Should it? We could probably make a stronger assertion about what should be printed to stdout here